README.md

```diff
@@ -84,15 +88,14 @@ The rendered output will be:
 
 ### 3\. Styling the Breadcrumbs
 
-The addon doesn't ship with default styling, this should be done inside the consuming project.
+The addon doesn't ship with default styling. This should be done inside the consuming project.
 
-### 4\. Arguments
+### 4\. Breadcrumbs component arguments
 
-Name              | Description               | Type   | Default
-:---------------- | :------------------------ | :----- | :------------------
-**className**     | The component class name. | String | 'breadcrumbs'
-**itemClassName** | The item class name.      | String | 'breadcrumbs__item'
-**linkClassName** | The link class name.      | String | 'breadcrumbs__link'
+Name          | Description                                                               | Type   
+:-------------| :------------------------------------------------------------------------ | :----- 
+**itemClass** | The class(es) that will be added to all child BreadcrumbItem components   | String 
```

Comment:
`BreadcrumbItem` => `BreadcrumbsItem` (or `<BreadcrumbsItem />`)

Reply:
Derp, will fix. Any preference?

Reply:
Either we use `<BreadcrumbsItem />` so it's the same like everywhere else. Or we use `BreadcrumbsItem` and update the rest of the BreadcrumbsItem mentions.

Reply:
Should be ok now